README.md

```diff
@@ -24,6 +24,8 @@ release. In the meantime, here is a little to get you started.
 
 ### Installing Brigade 2 on a _Private_ Kubernetes Cluster
 
+__Note: Brigade 2 is compatible only with Kubernetes versions 1.16.0+__
```

Comment:
I would remove "only" because having that there subtly implies a larger constraint than this really is. 1.16 is pretty old, so we shouldn't make a big deal out of this constraint.